workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -107,83 +168,49 @@ void OnReceived(object sender, DataReceivedEventArgs args)
         /// <summary>
         ///     Runs the redirected process and returns a task which can be waited on.
         /// </summary>
-        /// <param name="workingDirectory">The directory to run the command from.</param>
-        /// <param name="command">The filename to run.</param>
-        /// <param name="arguments">Parameters that will be passed to the command.</param>
         /// <param name="redirectStdout">Redirect standard output to Debug.Log</param>
         /// <param name="redirectStderr">Redirect standard error to Debug.LogError</param>
         /// <returns>A task which would return the exit code and output.</returns>
-        public static async Task<RedirectedProcessResult> RunInAsync(string workingDirectory, string command, string[] arguments, bool redirectStdout = false, bool redirectStderr = false)
+        public async Task<RedirectedProcessResult> RunAsync(bool redirectStdout = false, bool redirectStderr = false)
```

Comment:
`bool`-based APIs generally smell a bit. Prefer an `enum` as flags here instead. 
For example, `[Flags] enum { RedirectStdErr, RedirectStdOut, RedirectAll }`
This avoid call sites that look like `RunAsync(true, true) <-- huh?` or the verbose `RunAsync(redirectStdout: true, redirectStderr: true)`.
Also, as a general point, `bools` tend to multiply and encourage "customizing" functions that should really be split out into their own separate area of concern. I've seen monstrosities like `thing.DoUpdate(false, 7, true, true)` more times than I can count.

Reply:
Good point, let me rework this API